core/src/main/scala/org/apache/spark/serializer/serializers.scala

```diff
@@ -208,9 +209,9 @@ object ConnectionPropertiesSerializer
     output.writeString(connProps.url)
     output.writeString(connProps.driver)
     connProps.dialect match {
-      case GemFireXDDialect => output.writeByte(0)
-      case GemFireXDClientDialect => output.writeByte(1)
-      case SnappyDataClientPoolDialect => output.writeByte(3)
+      case SnappyDataDialect => output.writeByte(0)
```

Comment:
@sumwale a general comment just on the naming. The dialect which the Smart Connector should use theoretically should be the derby/gemxd dialect, specially from the executors, rather only from the executors when scanning tables. Do you think keeping the gemfirexd name for such access should be clearly mentioning that dialect as opposed to a client accessing SnappyData store, which will be primarily spark-sql dialect driven.

Reply:
Ok, can change this name to SnappyStoreDialect, SnappyStoreClientDialect and SnappyStorePoolDialect. Removing GemFireXD in the name will be good because we have lots of changes now over the erstwhile GemFireXD SQL support.

Reply:
ok.